### PR TITLE
completed assessment

### PR DESCRIPTION
**Pull request by Rethora**

### CODE REVIEW

File: code-challenges.test.js [hunk 1]
```diff
@@ -14,38 +14,114 @@
...
+// we will add each sentence to an array
+// we will return the array of sentences
+
+const storyTime = arr => {
+  return arr.map(val => {
+    const { name, occupation } = val
+    const capitalizedName = name.split(' ').map(n => n[0].toUpperCase() + n.slice(1)).join(' ')
+    return `${capitalizedName} is ${occupation}.`
+  })
+}
```
Comment (SunkissedQueen) [inline code-challenges.test.js:64]: Please include your test suite results

File: code-challenges.test.js [hunk 2]
```diff
@@ -14,38 +14,114 @@
...
 
 // b) Create the function that makes the test pass.
+
+// create a function named cubedSum
+// cubedSum will take in an array of numbers
+// we can loop over the array and cube each number and add it to a new array
+// we can then loop over that array and find the sum of all the cubed numbers
+// we can then return the sum
+
+const cubedSum = arr => arr.map(val => val ** 3).reduce((a, b) => a + b)
```
Comment (SunkissedQueen) [inline code-challenges.test.js:131]: Everything looks terrific. You know the one-liners are special to me. Please include your test suite results after all the functions are created. I feel that the functions passed. I just need to see that the test suite feels the same way.

File: interview-questions.md [hunk 3]
```diff
@@ -7,44 +8,54 @@ Then, research the question to expand on your answer. Even if you feel you have
```
Comment (SunkissedQueen) [inline interview-questions.md:8]: And that is exactly how you do interview questions. I can truly see that you understand the concepts and are willing to not only do research but also form your own understanding. Truly appreciate your thorough approach.

File: rails-commenting.rb [hunk 4]
```diff
@@ -7,24 +7,29 @@
 # FILE: app/controller/blog_posts_controller.rb
```
Comment (SunkissedQueen) [inline rails-commenting.rb:8]: Let's see....green check mark for your knowledge

File: code-challenges.test.js [hunk 5]
```diff
@@ -14,38 +14,114 @@
...
+    const people = [
+      { name: "ford prefect", occupation: "a hitchhiker" },
+      { name: "zaphod beeblebrox", occupation: "president of the galaxy" },
+      { name: "arthur dent", occupation: "a radio employee" }
+    ]
+    const peopleOutput = [
+      "Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."
+    ]
+
+    const peopleTwo = [
```
Comment (SunkissedQueen) [inline code-challenges.test.js:28]: Ok. Testing it on a middle name. Love it.

**Review by SunkissedQueen — CHANGES_REQUESTED** [review]
Dane, your assessments have been exciting to review. I appreciate the way you push yourself. I love seeing how you dive deeper and try different techniques with your coding. I just need you to add your test suite results to show if your function passed the test.

File: code-challenges.test.js (hunk 2 above)
Comment (Rethora) [reply code-challenges.test.js:131]: I'm sorry, that's my mistake. I have included the test results after each function creation as per request. Thank you all the feedback on my assessments over the weeks. I appreciate you Charlean!

**Review by SunkissedQueen — APPROVED** [review]
Thanks for updating

**Pull request merged**